src/dymaptic.GeoBlazor.Core/Extensions/Dimension.cs

```diff
@@ -0,0 +1,139 @@
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+
+namespace dymaptic.GeoBlazor.Core.Extensions;
+
+/// <summary>
+///     Defines a screen measurement in points or pixels. Supports implicit conversion from string, int, or double.
```

Comment:
There are a number of Dimension references in the JS API.  Confirming this dimension extension class lays the groundwork for the properties found in the DimensionSimpleStyle correct? https://developers.arcgis.com/javascript/latest/api-reference/esri-analysis-DimensionSimpleStyle.html

Reply:
Esri's use of the word `Dimension` in `DimensionSimpleStyle` and `DimensionAnalysis` seem to be focused on real-world dimensions, whereas this is for screen measurements. However, since they aren't using the exact same word as a type (so far), I think it's fine and there is no conflict.